app/components/primer/alpha/action_list/heading.rb

```diff
@@ -11,21 +11,27 @@ class Heading < Primer::Component
           :filled => "ActionList-sectionDivider--filled"
         }.freeze
         SCHEME_OPTIONS = SCHEME_MAPPINGS.keys.freeze
+        HEADING_MIN = 1
+        HEADING_MAX = 6
```

Comment:
Can you use the defined constants in static/constants.json?

Reply:
Ah, constants.json is generated from constants in the source code, so this is the right place for them to be.